_data/t.yml

```diff
@@ -266,3 +266,9 @@ SignatureFooter:
 PreRole:
   en: Formerly
   it: Precedentemente
+Team:
+  en: The Team
+  it: Il Team
+PastMembers:
+  en: People who have previously contributed to the Team
```

Comment:
```suggestion
  en: Previous contributors
```

?

Reply:
Questa me l'ha suggerita Daniela in persona, è lunga ma la lascerei così.